airflow/providers_manager.py

```diff
@@ -178,7 +181,49 @@ def _add_provider_info_from_local_source_file(self, path, package_name) -> None:
         except Exception as e:  # noqa pylint: disable=broad-except
             log.warning("Error when loading '%s': %s", path, e)
 
+    def _discover_extra_links(self) -> None:
+        """Retrieves all extra links defined in the providers"""
+        for provider_package, (_, provider) in self._provider_dict.items():
+            if provider.get("extra-links"):
+                for extra_link in provider["extra-links"]:
+                    self.__add_extra_link(extra_link, provider_package)
+
+    def __add_extra_link(self, extra_link_class_name, provider_package) -> None:
```

Comment:
Just want to double check`__` is intentional to assign it to class right?

Reply:
This makes it "private" -- python in the hood translates this to `__ProvidersManager_add_extra_link`

Reply:
Yup, just wanted to confirm though if that was intentional

Reply:
Yep. And on Ash's request I am changing it to protected.